thinkhazard/static/less/report.less

```diff
@@ -137,9 +137,23 @@ body {
 
 #map-drillup {
   position: absolute;
-  top: 10px;
-  left: 10px;
+  top: 5px;
+  left: 5px;
   z-index: 2;
+
+  background: rgba(255,255,255, 0.5);
+  border: 2px solid rgba(255,255,255,0.75);
+
+  &:hover {
+    background: rgba(255,255,255, 0.75);
+  }
+
+  i {
+    background: rgb(255,255,255);
+    color: #58595b;
+    padding: 3px 3.5px;
+    align-items: center;
```

Comment:
Is `align-items` really useful here?
I'd personally avoid too modern css tricks.